trackers/huggingface/app.py

```diff
@@ -21,10 +21,92 @@ def add_faq() -> None:
     """
     faq = Collapsable()
     faq.add_section(
-        "Why is this so empty?",
+        "How is MLAgility different from MLPerf?",
         (
-            "Because the FAQ of huggingface website still needs to be written. "
-            "We don't use the same FAQ as in our internal dashboard."
+            "Deep learning pioneers have been judging their progress with the Machine Learning "
+            "Performance (MLPerf) inference benchmark, but have found that the corpus of models "
+            "is small enough that it allows vendors to primarily compete by hand-optimizing "
+            "kernels. MLAgility offers a complementary approach to MLPerf by examining the "
+            "capability of vendors to provide turnkey solutions to a larger corpus of "
+            "off-the-shelf models. By providing a workflow that is representative of the "
+            "mass adoption customer on a variety of ML accelerators and effectively disallowing "
+            "hand-crafted kernels, MLAgility bridges the gap between MLPerf and the mass adoption "
+            "of hardware acceleration."
+        ),
+    )
+    faq.add_section(
+        "Why now for MLAgility?",
+        (
+            "Deep learning algorithms and their associated DL hardware accelerators are "
+            "transitioning from early adoption into mass adoption. Production DL is now "
+            "becoming available to the masses, with a desire to customize models to tackle "
+            "their specific problems, and then take the path of least resistance into "
+            "production. A market for turnkey solutions, starting with a model as input and "
+            "provision a cost- and latency-effective acceleration solution, often in the cloud, "
+            "as output, has emerged."
+        ),
+    )
+    faq.add_section(
+        "Which tool was used to generate those results?",
+        (
+            "All MLAgility results have been generated using the <b>benchit</b> tool, which is part "
+            "of the MLAgility Github Repository. You can learn more about it "
+            '<a href="https://github.com/groq/mlagility">here</a>.'
+        ),
+    )
+    faq.add_section(
+        "What is the experimental setup for each of the devices?",
+        [
+            "<b>x86</b>: Intel(R) Xeon(R) X40 CPU @ 2.00GHz on Google Cloud (custom: n2, 80 vCPU, 64.00 GiB) and OnnxRuntime version 1.14.0.",
+            "<b>nvidia</b>: NVIDIA A100 40GB on Google Cloud (a2-highgpu-1g) and TensorRT version 22.12-py3.",
+            "<b>groq</b>: GroqChip 1 on GroqFlow version 2.5.2 and GroqWare™ Suite version 0.9.2.",
```

Comment:
```suggestion
            "<b>groq</b>: GroqChip 1 on GroqFlow version 3.0.2 test package and GroqWare™ Suite version 0.9.2.",
```

That's what's in setup.py. Unless you were using older groqflow?